content/en/getting-started/help-and-support/index.md

```diff
@@ -10,89 +10,120 @@ cascade:
 
-## Dedicated support
+Community support is provided on a best-effort basis and is not guaranteed.
+We also encourage you to help others in the community by answering questions and sharing your experiences.
 
-Dedicated customer support is available to **Starter** & **Teams** plan users with an active subscription.
+### Standard support
 
-The support team is available to help you with:
+Standard support is available to all LocalStack users with an active Starter or Trial subscription.
+With standard support, you get access to 1v1 support based on best effort basis.
+With the best effort support, we do not provide fixed response times.
+However, as we understand the importance of timely assistance, our support team always strives to respond to inquiries and requests within 24–48 hours during our [regular business hours](#support-business-hours).
 
-- Troubleshooting LocalStack issues
-- Answering questions about LocalStack features and functionality
-- Providing guidance on how to use LocalStack
-- Providing guidance on how to integrate LocalStack with your application
+#### Troubleshooting Guidelines
 
-However, dedicated support does not include:
+In case you encounter any issues with LocalStack, follow these troubleshooting guidelines:
 
-- Support for AWS services
-- General troubleshooting of your application
-- Billing, account issues, or other questions that are not technical
+- Review the documentation and FAQs for possible solutions.
+- Check the system requirements to ensure compatibility.
+- Verify the configuration settings against the documentation.
+- Consult the error logs for any relevant error messages or warnings.
+- Seek guidance from the community through the user forum.
+
+{{< callout "note" >}}
+In many scenarios, we ask our customers to use Diagnosis endpoint to help us retrieve additional information.
+
+To use LocalStack's Diagnosis endpoint:
+
+- Set the environment variable `LS_LOG=trace`
+- Start LocalStack
+- Run the affected task(s)
+- Call the diagnostic endpoint `curl -s localhost:4566/_localstack/diagnose | gzip -cf > diagnose.json.gz` (Endpoint URL depends on your configuration)
+- Send the output file to our Slack chat
+
+Ensure that you avoid sending the diagnostic output to public channels or forums, as it may contain sensitive information.
+{{< /callout >}}
 
-If you have questions about billing, account issues, or other questions that are not technical, please contact us at [support@localstack.cloud](mailto:support@localstack.cloud) or through our [contact form](https://localstack.cloud/contact/).
+#### Channels
 
-You can avail dedicated support through the following channels:
+You can avail standard support through the following channels:
 
 - [LocalStack Slack Bot](https://localstack.cloud/slack)
 - [LocalStack Web Application](https://app.localstack.cloud)
 
-### LocalStack Slack Bot
+##### LocalStack Support Bot
 
 To create a support ticket:
```

Comment:
```suggestion
To create a support ticket:

- Search for the LocalStack Pro Support in our [Slack Community](https://localstack.cloud/slack).
- Navigate to Messages and type your request.
- Include relevant details such as error logs, configuration settings, and steps to reproduce the issue.

Providing detailed information upfront helps our team respond more quickly and efficiently.
After submitting, the support team will respond as soon as possible.
```

We should mention a list of what they need to provide do to resolve the issue more quickly.